tracer/src/Datadog.Trace/DiagnosticListeners/EndpointFeatureProxy.cs

```diff
@@ -12,7 +12,9 @@ namespace Datadog.Trace.DiagnosticListeners
     /// explicitly, e.g. by https://github.com/dotnet/aspnetcore/blob/v3.0.3/src/Servers/Kestrel/Core/src/Internal/Http/HttpProtocol.FeatureCollection.cs
     /// Also see AspNetCoreDiagnosticObserver.EndpointFeatureStruct
     /// </summary>
+#pragma warning  disable DDSEAL001 // Types should be sealed where possible. Types used for duck-typing cannot be sealed.
     internal class EndpointFeatureProxy
+#pragma warning restore DDSEAL001
```

Comment:
This is actually a (rare) `class` based duck type. We don't recommend these now (and can probably update to use `interface` instead, so just ignoring it for now